src/SB/Game/zNPCFXCinematic.cpp

```diff
@@ -460,7 +460,7 @@ void NCIN_BubSlam(const zCutsceneMgr*, NCINEntry* fxrec, S32 param)
     return;
     }
 
-    switch (fxrec->typ_ncinfx)
+   switch(fxrec->typ_ncinfx)
```

Comment:
Can you run the formatter on this file so things like this don't sneak into PRs?